alchemist/alchemist-swingui/src/main/kotlin/it/unibo/alchemist/boundary/gui/view/properties/RangedDoubleProperty.kt

```diff
@@ -125,12 +126,13 @@ class RangedDoubleProperty @JvmOverloads constructor(
         if (javaClass != other?.javaClass) return false
         other as RangedDoubleProperty
         if (name != other.name) return false
-        if (lowerBound != other.lowerBound) return false
-        if (upperBound != other.upperBound) return false
+        if (abs(lowerBound - other.lowerBound) < 0.0000001) return false
+        if (abs(upperBound - other.upperBound) < 0.0000001) return false
         return true
```

Comment:
This `if`/`return` chain is ugly. Can you refactor in a logical conjunction?
`fun ... = javaClass == other?.javaClass && name == other.name && ...`

Reply:
I'll wait for your answer on https://github.com/AlchemistSimulator/Alchemist/pull/115#discussion_r354496126 before doing this